tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/deep-config/test-suite-a/testbetanamespace-namespace/alphamember-variable.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <section>
+      <h1>alphaMember</h1>
+      <section>
+        <p><a href="/">Packages</a> > <a href="/test-suite-a/">test-suite-a</a> > <a href="/test-suite-a/testbetanamespace-namespace/">TestBetaNamespace</a> > <a href="/test-suite-a/testbetanamespace-namespace/alphamember-variable">alphaMember</a></p>
```

Comment:
nit: says TestBetaNamespace here but it probably should say alpha, not sure if this matters at all

Reply:
The namespace itself is `@beta`, but this inner member is `@alpha`.